website/docs/r/muting_schedule.html.markdown

```diff
@@ -91,6 +91,31 @@ resource "sumologic_muting_schedule" "muting_schedule" {
 }
 ```
 
+## Example Daily Muting Schedule With Group Supporting 
+
+```hcl
+resource "sumologic_muting_schedule" "muting_schedule" {
+  name = "Muting Schedule For one time"
+  description = "This is an example for one time Muting schedule for all monitor"
```

Comment:
Please change the description to reflect the configuration. The schedule has rrule and monitor ID defined.

Reply:
Updated.